### PR TITLE
Jag/daemonaddpoolentry

### PR DESCRIPTION
**Pull request by Joelgranados**
This sets us up to merge PR#16 which adds an additional element to the pool_entry struct

### CODE REVIEW

File: src/flexalloc_mm.c [hunk 1]
```diff
@@ -734,6 +734,11 @@ fla_print_pool_entry(struct flexalloc *fs, struct fla_pool_entry * pool_entry)
   uint32_t tmp;
   fprintf(stderr, "===============\n");
   fprintf(stderr, "Pool Entry %p\n", pool_entry);
+  fprintf(stderr, "obj_nlb : %"PRIu32"\n", pool_entry->obj_nlb);
```
Comment (jwdevantier) [inline src/flexalloc_mm.c:737]: Not for this PR, but I'd like for functions like these to get removed in favor of some code in the python bindings that lists/dumps debug information.

**Review by jwdevantier — APPROVED** [review]
looks good :)

BONUS POINTS if you create some smaller struct for holding the `fla_pool_entry` metadata only, such that we don't end up sending the full string name back and forth (it's by far the largest bit of metadata and adds to the overhead of the protocol).

File: src/flexalloc_mm.c (hunk 1 above)
Comment (Joelgranados) [reply src/flexalloc_mm.c:737]: I actually like these ;). I use them a lot to debug. sometimes I place print statements before and after a call to see how they change. Like in the case of this PR, I printed it in the daemon and then I printed it in the client.
I have a question though:
Is there a way to inject a "print pool_entry" call from python into any place in the C code?

**Comment by Joelgranados** [comment]
> looks good :)
> 
> BONUS POINTS if you create some smaller struct for holding the `fla_pool_entry` metadata only, such that we don't end up sending the full string name back and forth (it's by far the largest bit of metadata and adds to the overhead of the protocol).

I have added a TODO for this.

**Pull request merged**